src/Facebook/InstantArticles/Transformer/Getters/GetterFactory.php

```diff
@@ -8,8 +8,6 @@
  */
 namespace Facebook\InstantArticles\Transformer\Getters;
 
-use Facebook\InstantArticles\Transformer\Validators\Type;
```

Comment:
The class [`Facebook\InstantArticles\Validators\Type`](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/master/src/Facebook/InstantArticles/Validators/Type.php) exists, but this import is never used anyway.